Config Example Files Updates (#1215)
cmss13-devs/cmss13: config/example/role_whitelist.txt

> #This document should be placed in Config to have effect. Lines starting with # are comments

Reviewer: ```suggestion
# This document should be placed in /config/ to have effect. Lines starting with # are comments
```

# The format of this document is the person's ckey followed by their role permissions.
# You must list all permissions for a single ckey using one line.

# Casing does not matter, spacing does not matter, and underscores do not matter.
# +Y_aut j a El_DER will be converted into yautjaelder.
# You may add any of the possible roles below, using different names as indicated.
# Adding two or more of the same role will not matter.
# Council and Legacy roles give the general WL too. LegacyRank is not above Leader (Senator)

> # Elder rank means Clan leader for  Yauja

Reviewer: ```suggestion
# Elder means Clan Leader for Yautja
```

> # SEA flag is usually not necessay as it is given though the various staff/mentor ranks for now, but this may chage in the future.

Reviewer: ```suggestion
# SEA flag is unnecessary as it is given though the various staff/mentor ranks
```

######################################################
# YAUTJA FLAGS
# +Yautja
# +YautjaElder
# +YautjaCouncil
# +YautjaLeader
# +YautjaCouncilLegacy
# COMMANDER FLAGS
# +Commander
# +CommanderCouncil
# +CommanderLeader
# +CommanderCouncilLegacy
# SYNTHETIC FLAGS
# +Synthetic
# +SyntheticCouncil
# +SyntheticLeader

> # MENTOR FLAGS (this is just the SEA half not the mhelps half)

Reviewer: ```suggestion
# MENTOR FLAGS (for access to the SEA role, not MHelps, MentorSay, etc)
```
